### PR TITLE
Vp32xx GPIO config

### PR DESCRIPTION
**Pull request by filipleple**

### CODE REVIEW

File: src/mainboard/protectli/vault_adl_n/Kconfig [hunk 1]
```diff
@@ -0,0 +1,86 @@
+if BOARD_PROTECTLI_VP32XX
+
+config BOARD_SPECIFIC_OPTIONS
+	def_bool y
+	select BOARD_ROMSIZE_KB_16384
+	select SOC_INTEL_ALDERLAKE_PCH_N
+	select SOC_INTEL_COMMON_BLOCK_HDA_VERB
+	select SOC_INTEL_ENABLE_USB4_PCIE_RESOURCES
```
Comment (miczyg1) [inline src/mainboard/protectli/vault_adl_n/Kconfig:8]: THese resources are actually not needed on ADL-N as the SoC does not have USB4/Thunderbolt4

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: src/mainboard/protectli/vault_adl_n/Kconfig (hunk 1 above)
Comment (filipleple) [reply src/mainboard/protectli/vault_adl_n/Kconfig]: [Removed](https://github.com/Dasharo/coreboot/pull/567/commits/918da71b70c8cbf82572e81f7dab4b0242af7170)

File: src/mainboard/protectli/vault_adl_n/Kconfig [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+	select SUPERIO_ITE_IT8659E
+	select DRIVERS_UART_8250IO
+	select DRIVERS_I2C_GENERIC
+	select DRIVERS_INTEL_PMC
```
[the remark this answers is not in the document]
Comment (filipleple) [reply src/mainboard/protectli/vault_adl_n/Kconfig]: [Removed](https://github.com/Dasharo/coreboot/pull/567/commits/918da71b70c8cbf82572e81f7dab4b0242af7170)

File: src/mainboard/protectli/vault_adl_n/gpio.c [hunk 3]
```diff
@@ -0,0 +1,1585 @@
...
+	/* DW0: 0x00000700, DW1: 0x00000000 */
+	/* DW0: PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPD_DRAM_RESETB, NONE, PWROK, NF1), */
+	_PAD_CFG_STRUCT(GPD_DRAM_RESETB, PAD_FUNC(NF1) | PAD_BUF(TX_RX_DISABLE), 0),
+
+	/* ------- GPIO Group PCIe vGPIO ------- */
+
+	/* GPIO */
+
+	/* GPIO */
```
[the remark this answers is not in the document]
Comment (mkopec) [reply src/mainboard/protectli/vault_adl_n/gpio.c:938]: looks like a mismatch in vGPIO pin names between inteltool and intelp2m

[the remark this answers is not in the document]
Comment (filipleple) [reply src/mainboard/protectli/vault_adl_n/gpio.c:938]: @mkopec  Hmm, unfortunately neither replacing the vGPIO in inteltool log with VGPIO, nor replacing VGPIO with vGPIO in intelp2m and rebuilding seems to change anything.

What I've noticed is that inteltool output looks a bit different for e.g. NV4xMB:

```
0x0900: 0x0000000000000000 GPP_vGPIO_PCIE_0 GPIO
0x0910: 0x0000000000000000 GPP_vGPIO_PCIE_1 GPIO
0x0920: 0x0000000000000000 GPP_vGPIO_PCIE_2 GPIO
0x0930: 0x0000000000000000 GPP_vGPIO_PCIE_3 GPIO
```

vs

```
0x07f0: 0x0000000080000400 VGPIO_PCIE_0 VGPIO_PCIE_0
0x0800: 0x0000000080000400 VGPIO_PCIE_1 VGPIO_PCIE_1
0x0810: 0x0000000080000400 VGPIO_PCIE_2 VGPIO_PCIE_2
0x0820: 0x0000000080000400 VGPIO_PCIE_3 VGPIO_PCIE_3
```

So I've also tried `sed -i 's/GPP_vGPIO_PCIE/VGPIO_PCIE/g' inteltool.log`, which only yields a few less of the `/* GPIO */` lines.

**Comment by miczyg1** [comment]
Superseded by https://github.com/Dasharo/coreboot/pull/574

**Pull request closed without merging**